components/AppFooter.vue

```diff
@@ -1,5 +1,5 @@
 <script setup lang="ts">
-import { ProviderName } from '@lukso/web-components/dist/components/lukso-share'
+import type { ProviderName } from '@lukso/web-components'
```

Comment:
Why do we need this? Import type is not really a thing anymore is it?

Reply:
Apparently for Nuxt it still is:

> Vue requires that type imports be explicit (so that the Vue compiler can correctly optimise and resolve type imports for props and so on)

Reply:
That's insane, ok